traffic_ops/testing/api/v2/tc-fixtures.json

```diff
@@ -31,6 +31,13 @@
             "shortName": "pg1",
             "typeName": "MID_LOC"
         },
+        {
```

Comment:
These additions should be made to the v3 fixtures instead now

Reply:
Moved to v3 fixtures in ba4d584054.